### PR TITLE
Name change for the friendly space carps

### PR DESCRIPTION
**Pull request by ASmallCuteCat**
Changes the name of "tame space carp" to "domesticated space carp", to reflect what it says in their description.

(i also found out that there was an actual difference between 'tame' and 'domesticated'. the more you know)

### CODE REVIEW

**Comment by ASmallCuteCat** [comment]
!review

**Review by alsoandanswer — CHANGES_REQUESTED** [review]
The description should be changed to tame rather than the other way around. A chicken is domesticated, a lion is tamed.

**Comment by ASmallCuteCat** [comment]
> The description should be changed to tame rather than the other way around. A chicken is domesticated, a lion is tamed.

Easy change - though I should ask, since the science sampler has unique text for the non-bitey carps: I don't remember the exact wording, but the line goes something like "has genetic markers for docile behavior".

Should that line be changed/taken out as well? I can do a little re-writing so that there's some consistency between description, name, and science sampler text.

**Review by alsoandanswer — APPROVED** [review]
No, this is sufficient. Docile basically means non-aggressive, which is about its behavior rather than its relationship to man.

File: code/modules/mob/living/simple_animal/friendly/carp.dm [hunk 1]
```diff
@@ -1,5 +1,5 @@
 /mob/living/simple_animal/carp
-	name = "tame space carp"
+	name = "domesticated space carp"
```
Comment (alsoandanswer) [inline code/modules/mob/living/simple_animal/friendly/carp.dm:2]:
```suggestion
	name = "tame space carp"
```

File: code/modules/mob/living/simple_animal/friendly/carp.dm [hunk 2]
```diff
@@ -1,5 +1,5 @@
...
 	desc = "A domesticated, floating space carp. Careful around the teeth."
```
Comment (alsoandanswer) [inline code/modules/mob/living/simple_animal/friendly/carp.dm:3]:
```suggestion
	desc = "A tame, floating space carp. Careful around the teeth."
```

**Review by alsoandanswer — CHANGES_REQUESTED** [review]

**Review by alsoandanswer — APPROVED** [review]

**Review by NonQueueingMatt — APPROVED** [review]

**Pull request merged**